packages/manager/apps/dedicated/client/app/components/move-ip/index.js

```diff
@@ -0,0 +1,22 @@
+import angular from 'angular';
+
+import 'angular-translate';
+import '@ovh-ux/ui-kit';
+import '@ovh-ux/ng-translate-async-loader';
+import ngAtInternet from '@ovh-ux/ng-at-internet';
+
+import component from './move-ip.component';
+
+const moduleName = 'ovhManagerDedicatedMoveIp';
+
+angular
+  .module(moduleName, [
+    ngAtInternet,
+    'ngTranslateAsyncLoader',
```

Comment:
the module is not lazy-loaded, do you need this?

Reply:
@radireddy This is the module that allows to inject translations, it is required whether the module is lazyloaded or not